akka-docs/src/main/paradox/typed/actor-lifecycle.md

```diff
@@ -46,7 +46,7 @@ is a tool that mimics the old style of starting up actors.
 
 ### Spawning Children
 
-Child actors are spawned with @scala[@unidoc[akka.actor.typed.scaladsl.ActorContext]]@java[@unidoc[akka.actor.typed.javadsl.ActorContext]]'s `spawn`. 
+Child actors are spawned with @apidoc[typed.*\.ActorContext]'s `spawn`.
```

Comment:
this works? nice!

Reply:
once lightbend/sbt-paradox-apidoc#15 is merged it will :smile: